ARCHITECTURE.md

```diff
@@ -108,6 +108,18 @@ This table keeps track of the manifests of the validators.
 | `revoked`            |Whether the manifest has been revoked.                      |
 | `seq`                |The sequence number of this manifest.                       |
 
+### `amendments_enabled`
+
+This table keeps track of the amendments enabled on each network.
+
+| Key                  | Definition                                               |
+|----------------------|----------------------------------------------------------|
+| `amendment_id`       |The amendment id.                                         |
+| `networks`           |The network where the amendment has been enabled.         |
+| `ledger_index`       |The ledger where the amendment has been enabled.          |
+| `tx_hash`            |The transaction hash where the amendment has been enabled.|
+| `date`               |The date and time when the amendment has been enabled.    |
```

Comment:
Is this in ripple epoch time or POSIX time?

Reply:
It's converted to js `Date` type instead of a number so I'm not sure it needs clarification here (it will show YYYY-MM-DD HH:MM:SS in the database).

Reply:
I think it's still important to mention because that changes how you interpret the number. (The `Date` type is how it's formatted, but knowing if it's ripple epoch changes what that date represents. If you directly change a ripple epoch number into a Date, it'll give you the wrong date)

Reply:
It'd probably be easier to store it in ripple epoch, tbh. You can convert when outputting.